<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Suchiz Rendering Engine</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QSplitter" name="sceneSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="sceneTab">
        <attribute name="title">
         <string>Scene</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QSplitter" name="buttonSplitter">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QWidget" name="layoutWidget">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="sceneOptionsLabel">
               <property name="text">
                <string>- Scene Options -</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="demoButton">
               <property name="text">
                <string>Demo Scene</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="clearButton">
               <property name="text">
                <string>Clear Scene</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="moveLightCheckBox">
               <property name="text">
                <string>Move light</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="layoutWidget_2">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="sceneManagerLabel">
               <property name="text">
                <string>- Scene Manager -</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListWidget" name="objectListView"/>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QPushButton" name="editButton">
                 <property name="text">
                  <string>Edit</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="deleteButton">
                 <property name="text">
                  <string>Delete</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="addButton">
               <property name="text">
                <string>+</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="animationTab">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <attribute name="title">
         <string>Animation</string>
        </attribute>
       </widget>
      </widget>
      <widget class="QWidget" name="sceneContainer" native="true"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
